Sources/KingfisherManager.swift

```diff
@@ -159,6 +163,21 @@ public class KingfisherManager {
         return retrieveImageWithResource(Resource(downloadURL: URL), optionsInfo: optionsInfo, progressBlock: progressBlock, completionHandler: completionHandler)
     }
     
+    /**
+     Determine if a cached image exists for the given image, as keyed by the URL. It will return true if the
+     image is found either in memory or on disk. Essentially as long as there is a cache of the image somewhere
+     true is returned.
+     
+     - parameter url: The image URL.
+     
+     - returns: True if the image is cached, false otherwise.
+     */
+    public func cachedImageExistsforURL(url: NSURL) -> Bool {
```

Comment:
Although it could be useful and a quick way to check an image cached or not, I don't think it's a good idea to have a method for it here. 

I believe it should only in the `ImageCache`. Furthermore, since `cache` of `KingfisherManager` could be set by the users, it is not clear and might be confusing which cache is used for the checking.

Generally speaking, I cannot agree to add this here. We could use the same methods in `ImageCache` directly. Would you mind to remove this before this p-r could be merged?